File: JAVA/count_fact.java
import java.util.*;
class count_fact{
    public static void main(String[] args) {
        int count=0,i,n;
        System.out.println("Enter a number to find its factors:");
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        System.out.println("The factors of " + n + " are:");
        for(i=1; i<=n; i++) {
            if(n % i == 0) {
                System.out.print(i + " ");
                count+=1;
                System.out.println("Count of factors: " + count);
            }
            sc.close();
        }
    }
}

File: Python/count_fact.py
import math
n = int(input("Enter a number to find its factors: "))
count = 0
sqrt_n = int(math.sqrt(n))
for i in range(1, sqrt_n + 1):
    if n % i == 0:
        if i == n // i:
            count += 1
        else:
            count += 2

print("The count of factors is:", count)
